### PR TITLE
Comp Rewards | [Tests]

### PR DESCRIPTION
**Pull request by iri031**
Test cases for comet rewards connector. 
P.S. Currently the rewards are not live or there are zero rewards owed.

- [x] Review
- [x] Deployment [here](https://github.com/Instadapp/dsa-connectors/pull/262#issuecomment-1240023562)
- [x] Add to SDK https://github.com/Instadapp/dsa-connect/pull/229
- [x] Enable
- [ ] Add to docs

### CODE REVIEW

**Comment by iri031** [comment]
Screenshot from 2022-09-06 14-00-34

File: contracts/mainnet/connectors/compound/v3-rewards/main.sol [hunk 1]
```diff
@@ -17,55 +17,48 @@ abstract contract CompoundV3RewardsResolver is Events, Helpers {
...
 	 * @param setId ID stores the amount of rewards claimed.
 	 */
-	function claimRewardsTo(
+	function claimRewardsOnBehalfOf(
 		address market,
-		address account,
-		address dest,
-		bool accrue,
+		address owner,
+		address manager,
```
Comment (thrilok209) [inline contracts/mainnet/connectors/compound/v3-rewards/main.sol]: @Richa-iitr I think we can change it back `to`.
```suggestion
		address to,
```

File: contracts/mainnet/connectors/compound/v3-rewards/events.sol [hunk 2]
```diff
@@ -7,16 +7,14 @@ contract Events {
...
 	);
 
-	event LogRewardsClaimedTo(
+	event LogRewardsClaimedOnBehalf(
 		address indexed market,
-		address indexed account,
-		address to,
+		address indexed owner,
+		address manager,
 		uint256 indexed totalClaimedInWei,
```
Comment (thrilok209) [inline contracts/mainnet/connectors/compound/v3-rewards/events.sol]:
```suggestion
		uint256 indexed rewardsClaimed,
```

File: contracts/mainnet/connectors/compound/v3-rewards/main.sol (hunk 1 above)
Comment (iri031) [reply contracts/mainnet/connectors/compound/v3-rewards/main.sol]: done

File: contracts/mainnet/connectors/compound/v3-rewards/events.sol (hunk 2 above)
Comment (iri031) [reply contracts/mainnet/connectors/compound/v3-rewards/events.sol]: done

**Comment by iri031** [comment]
## Deployment V3-Rewards

- **Name**: `COMPOUND-V3-REWARDS-A`
- **Addr**: [0x44aC5eD26d8B7BBcF3Bcb9Ac486191cBb294B140](https://etherscan.io/address/0x44aC5eD26d8B7BBcF3Bcb9Ac486191cBb294B140#code)
- **tx**:  [here](https://etherscan.io/tx/0x92baf866da2f7981dd03cf4721429bd157126118c3bbe0a1c38ce71da70106e9)

**Review by pradyuman-verma — APPROVED** [review]

**Pull request merged**